scripts/a2_collections

```diff
@@ -137,3 +142,97 @@ authLoad() {
     fi
   done
 }
+
+#############
+# authGen
+#############
+# Purpose:
+#     Generate a list of a2 auth resources.
+#     This generates dummy data that is not particularly realistic
+#     but can be used to observe general performance characteristics.
+#     NB: When generating policies, it builds one policy per token,
+#     so you should also generate a set of tokens with the same parameters.
+#     This function also allows you to delete your generated resources just as easily.
+# 
+# Example:
+#     Create 500 policies starting with a seed of 101:
+#         $ authGen policies create 500 101
+#     Per above note, those policies need these members:
+#         $ authGen tokens create 500 101
+#     Now, just for illustration, delete the last 100 of those 500 policies:
+#         $ authGen policies delete 100 501
+#
+# Options:
+#    -2 or --v2           use IAM V2 endpoints
+#
+# TODO: Except for common things (e.g. teams) this supports only IAM v1 so far.
+#
+authGen() {
+  # process options
+  auth_path="api/v0/auth"
+  while [[ "$1" =~ ^- ]]; do
+    case "$1" in
+      -2|--v2) auth_path="apis/iam/v2beta"; shift;;
+      -*) echo "'$1' unknown"; return 1;;
+    esac
+  done
+
+  # prerequisites
+  ([ -z "$TOK" ] || [ -z "$TARGET_HOST" ]) && { echo "Need TOK and TARGET_HOST variables"; return; }
+  [ -z "$4" ] && { echo "usage: authGen [ <options> ] <resource> <mode> <count> <seed>"; return; }
+  if ! type jo &> /dev/null
+  then
+    echo "Need jo installed"; return;
+  fi
+  RESOURCE=$1
+  MODE=$2
+  COUNT=$3
+  SEED=$4
+  ID_PREFIX=test-$RESOURCE
+  [[ ! $MODE =~ ^(create|delete)$ ]] && { echo "mode must be 'create' or 'delete'"; return; }
+  [[ ! $RESOURCE =~ ^(tokens|users|teams|policies)$ ]] && { echo "resource must be in: tokens, users, teams, policies"; return; }
+
+  if [[ $MODE == create ]]; then
+    echo "Creating $COUNT $RESOURCE..."
+    for (( i = 0; i < $COUNT; i++ )) 
+    do
```

Comment:
Since this is dev code, I think it is fine as is, but you might consider extracting the body of this for loop into a `create_resource` function. Similarly, a delete_resource function for the loop in the else clause.